Gemfile

```diff
@@ -4,11 +4,16 @@ gemspec
 unless ENV['CUCUMBER_USE_RELEASED_CORE']
   core_path = File.expand_path("../../cucumber-ruby-core", __FILE__)
   wire_path = File.expand_path("../../cucumber-ruby-wire", __FILE__)
-  if File.exist?(core_path) && !ENV['CUCUMBER_USE_GIT_CORE']
-    gem 'cucumber-core', :path => core_path
-    gem 'cucumber-wire', :path => wire_path
+  if File.exist?(core_path) && !ENV["cucumber_use_git_core"]
+    gem "cucumber-core", :path => core_path
   else
-    gem 'cucumber-core', :git => "https://github.com/cucumber/cucumber-ruby-core.git"
+    gem "cucumber-core", :git => "git://github.com/cucumber/cucumber-ruby-core.git"
```

Comment:
Oh jeez, can't believe I didn't notice that. Thanks for pointing that out - will change.